tests/kafkatest/services/monitor/jmx.py

```diff
@@ -110,7 +110,7 @@ def read_jmx_output(self, idx, node):
                 object_attribute_names = line.strip()[1:-1].split("\",\"")[1:]
                 continue
             stats = [float(field) for field in line.split(',')]
-            time_sec = int(stats[0]/1000)
+            time_sec = int(stats[0]/self.report_interval)
```

Comment:
If the reporting interval itself is increase, `stats[0]` would just have a different value, but this conversion from `ms` to `sec` should still be `/1000` if I understand the code correctly?